autoload/neomake/makers/ft/scala.vim

```diff
@@ -1,11 +1,23 @@
 " vim: ts=4 sw=4 et
 function! neomake#makers#ft#scala#EnabledMakers()
-    let makers = ['scalac']
-    if neomake#utils#Exists('scalastyle')
+    let makers = get(g:, 'neomake_scala_fast', 0) ? ['fsc'] : ['scalac']
+    if neomake#utils#Exists('scalastyle') && get(g:, 'neomake_scala_style', 1)
```

Comment:
Hmm. Any harm in running both? If that won't work, then a configuration variable is ok, but be sure to add an entry for it in the docs.